graylog2-web-interface/src/components/pipelines/StageForm.jsx

```diff
@@ -93,11 +90,6 @@ const StageForm = createReactClass({
 
   _saved() {
```

Comment:
For the saving if I override a stage number I'm still able to save. there is small part of the code is master that dealt with checking `isOverridingStage` that is missing here.

Reply:
Good catch! I will fix it 👍